app/styles/globals.css

```diff
@@ -76,29 +76,32 @@ i {
 }
 
 table {
+  line-height: 1.3;
```

Comment:
We should transition over to `line-height: 1.3` across the entire webapp (since it looks much much better), but that's a lot of work, so I'm doing it in chunks.